lib/profile/updates.js

```diff
@@ -9,14 +9,19 @@ module.exports = function (log) {
   return function start(messageQueue, push) {
 
     function handleProfileUpdated(message) {
+      let uid
```

Comment:
@jrgm why not something like `const uid = message && message.uid` to avoid unnecessary `let`?